medusa/medusacli.py

```diff
@@ -251,14 +253,15 @@ def restore_cluster(medusaconfig, backup_name, seed_target, temp_dir, host_list,
               multiple=True, default={})
 @click.option('--use-sstableloader', help='Use the sstableloader to load the backup into the cluster',
               default=False, is_flag=True)
+@click.option('--version-target', help='Target Cassandra version', required=False, default="3.11.9")
 @pass_MedusaConfig
 def restore_node(medusaconfig, temp_dir, backup_name, in_place, keep_auth, seeds, verify, keyspaces, tables,
-                 use_sstableloader):
+                 use_sstableloader, version_target):
     """
     Restore single Cassandra node
     """
     medusa.restore_node.restore_node(medusaconfig, Path(temp_dir), backup_name, in_place, keep_auth, seeds,
-                                     verify, set(keyspaces), set(tables), use_sstableloader)
+                                     verify, set(keyspaces), set(tables), use_sstableloader, version_target)
```

Comment:
Heads up you forgot to add this new arg in the corresponding method in `restore_node.py`